Assets/GothicVR/Scripts/Properties/DoorProperties.cs

```diff
@@ -0,0 +1,20 @@
+using ZenKit.Vobs;
+
+namespace GVR.Properties
+{
+    public class DoorProperties : InteractiveProperties
```

Comment:
Nice you're setting these already. I need them for the chest/door opening stuff later. 👍